core/src/main/java/org/kiwix/kiwixmobile/core/settings/SettingsContract.kt

```diff
@@ -15,17 +15,13 @@
  * along with this program. If not, see <http://www.gnu.org/licenses/>.
  *
  */
+package org.kiwix.kiwixmobile.core.settings
 
-package org.kiwix.kiwixmobile.core.settings;
+import org.kiwix.kiwixmobile.core.base.BaseContract
 
-import org.kiwix.kiwixmobile.core.base.BaseContract;
-
-interface SettingsContract {
-  interface View extends BaseContract.View<Presenter> {
-
-  }
-
-  interface Presenter extends BaseContract.Presenter<View> {
-    void clearHistory();
+internal interface SettingsContract {
```

Comment:
Making it `internal`,we can't access in our other modules such as `kiwix app` and `custom`

Reply:
Done

Reply:
@MohitMaliDeveloper I don't see any new code.

Reply:
hi @gouri-panda,
changes done